dask/dataframe/core.py

```diff
@@ -1520,12 +1511,16 @@ def elemwise(op, *args, **kwargs):
     dsk = dict(((_name, i), (op2,) + frs) for i, frs in enumerate(zip(*keys)))
     dsk = merge(dsk, *[d.dask for d in dasks])
 
-    if columns is not None:
-        return DataFrame(dsk, _name, columns, divisions)
-    else:
-        column_name = name or consistent_name(n for df in dfs
-                                              for n in df.columns)
-        return Series(dsk, _name, column_name, divisions)
+    if columns == no_default:
+        if len(dfs) >= 2 and len(dasks) != len(dfs):
+            # should not occur in current funcs
+            msg = 'elemwise with 2 or more DataFrames and Scalar is not supported'
+            raise NotImplementedError(msg)
```

Comment:
Are there no element-wise operations that affect more than two operands?  What is the limitation here?